src/ngraph/frontend/onnx_import/op/matmul.cpp

```diff
@@ -132,9 +159,24 @@ namespace ngraph
                     for (std::size_t g = 0; g < groups; ++g)
                     {
                         const auto& sliced_left = get_sub_matrix(left, g);
-                        const auto& sliced_right = get_sub_matrix(right, g);
-
-                        auto sub_dot = std::make_shared<ngraph::op::Dot>(sliced_left, sliced_right);
+                        std::shared_ptr<ngraph::Node> sliced_right = get_sub_matrix(right, g);
+
+                        std::shared_ptr<ngraph::Node> sub_dot;
+
+                        if (quantized)
+                        {
+                            sliced_right = std::make_shared<ngraph::op::Reshape>(
```

Comment:
Please make single _big_ reshape on whole data after this `if` 
https://github.com/NervanaSystems/ngraph/blob/377da3017239233c1813bf2bce6a94f9157ed555/src/ngraph/frontend/onnx_import/op/matmul.cpp#L144-L147